CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All Notable changes to `schema-org` will be documented in this file.
 
+## 5.0.0 - 2021-10-?
+- Upgrade to schema.org:v11.01
+
+## 4.0.0 - 2021-10-?
```

Comment:
Hopefully GH will updated this one when #168 is updated to have a data and merged in.